AzureTable/Microsoft.DataTransfer.AzureTable/Defaults.cs

```diff
@@ -43,6 +43,8 @@ public static void SetCurrent(IDefaults defaults)
         private sealed class LibraryDefaults : IDefaults
         {
             public AzureTableInternalFields SourceInternalFields { get { return AzureTableInternalFields.All; } }
+
+            public AzureTableLocationMode SourceLocationMode { get { return AzureTableLocationMode.PrimaryOnly; } }
```

Comment:
I think this should return `null` to be consistent with Azure Storage SDK default value.